### DISCUSSION TITLE
How can I refer request query parameters in response?

### DISCUSSION THREAD

**Discussion by c0d3rb4b4** [stoplightio/prism#2585] [Q&A]
I want my mock response to include my request query parameter. For example, my OAS looks like:

```
openapi: 3.0.0
info:
  title: Account
  version: 0.0.0
paths:
  /account:
    get:
      parameters:
        - in: query
          name: AccountId
          schema:
            type: string
          description: the accountId
      responses:
        '200':
          description: Success Response
          content:
            application/json:
              example:
                Response:
                  Account:
                  - AccountId: $request.query.AccountId
                    CompanyId: '11221122'
```

I used the OpenAPI [runtime expressions](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#runtime-expressions) syntax in the example response for AccountId, which is used in Prism's [callback support](https://github.com/stoplightio/prism/blob/master/examples/callbacks/payment-service.oas3.yaml), but it doesn't work here.

Is there any other way this functionality is supported?

**Comment by kaos** [comment]
I'm looking for the same use case. I have schema mocking an endpoint like `GET /foo/<id>` and I'd like the returned object to use this `id` rather than a hard coded or dynamically faked id.
